tools/SetupFlow/DevHome.SetupFlow/Windows/DevDriveWindow.xaml

```diff
@@ -0,0 +1,17 @@
+<windows:SecondaryWindow
+    x:Class="DevHome.SetupFlow.Windows.DevDriveWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:windows="using:DevHome.Common.Windows"
+    IsTopLevel="True"
+    MinHeight="518"
+    Height="518"
+    MinWidth="577"
+    Width="577"
+    mc:Ignorable="d">
+    <windows:SecondaryWindow.WindowTitleBar>
+        <windows:WindowTitleBar />
+    </windows:SecondaryWindow.WindowTitleBar>
```

Comment:
@bbonaby I kept DevDrive file structure as is (Window file and View file separate). But now we can (optionally) add the view content directly in the `DevDriveWindow` XAML. Alternatively (current solution), we can set the window content by calling the `base(devDriveView)` constructor